src/V3LinkCells.cpp

```diff
@@ -699,10 +689,59 @@ class LinkCellsVisitor final : public VNVisitor {
     }
 };
 
+class LinkConfigsVisitor : public VNVisitor {
```

Comment:
This is done before the main LinkCellsVisitor, so put this whole class before that class.